### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by hanneshayashi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# appe - Alerting Policy Price Estimator
+
+Starting January 2025, Google will begin charging for [Alerting Policies](https://cloud.google.com/monitoring/alerts).
+
+While Google has provided [documentation and examples](https://cloud.google.com/stackdriver/pricing#pricing-alerting), it is still very hard to actually estimate the cost of an Alerting Policy, let alone if you have dozens or possible hundreds of them.
+Using `appe`, you can easily estimate the price of not just an individual Alerting Policy, but all of them in your entire organization using a single command.
+
+## Required Permissions
```
Comment (apgiorgi) [inline README.md:17]: Can we add a TL;DR for those who prefer skipping the (excellent) long description?

File: README.md
Comment (apgiorgi) [inline README.md:1]: I would include a "How to build/install" section, or "where to download", etc.

Comment (hanneshayashi) [reply README.md:1]: Added in https://github.com/doitintl/gcp-tool-appe/pull/1/commits/9b0d7d8688a12f8f337467de0a8ce301ecb8d49a

**Review by apgiorgi — APPROVED** [review]
Great work! Thanks Hannes!

**Pull request merged**